### ISSUE TITLE
Implement glance web_download import method for images

### ISSUE THREAD

**Issue by freddebacker** [terraform-providers/terraform-provider-openstack#887]
Hi there,

I'm planning to implement a new feature. I already have some working code, but waiting for pending gophercloud PR before submitting.

### Terraform Version
Terraform v0.12.8
+ provider.openstack v1.23.0

### Affected Resource(s)
- openstack_images_image_v2

### Terraform Configuration Files
```hcl
provider "openstack" {
}

resource "openstack_images_image_v2" "image" {
  name             = "test"
  image_source_url = "https://releases.rancher.com/os/latest/rancheros-openstack.img"
  container_format = "bare"
  disk_format      = "qcow2"
}
```

### Desired Behavior
I want to be able to instruct openstack to download images directly from remote source using the "web-download" import method: https://docs.openstack.org/api-ref/image/v2/#the-web-download-import-method 

### Actual Behavior
The current implementation downloads all images to a local cache folder and uploads them from there. This is something I want to avoid because the amount and size of images I want to upload.

### References
- [x] Pending: https://github.com/gophercloud/gophercloud/pull/1725

**Comment by ssolkhon** [comment]
Hey, 

looks like the referenced PR is merged. :). +1 for this feature. 
Have you made any more progress with this?

Thanks,
Scott

**Comment by freddebacker** [comment]
@ssolkhon,

I'm still stuck because of the failing CI test, which I believe to be caused by the specifics of the OpenLab environment.

Fred

**Comment by freddebacker** [comment]
merged to master for v1.27.0
